SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -358,7 +360,8 @@ void USpatialReceiver::ReceiveActor(Worker_EntityId EntityId)
 		}
 	}
 
-	if (AActor* EntityActor = Cast<AActor>(PackageMap->GetObjectFromEntityId(EntityId)))
+	AActor* EntityActor = Cast<AActor>(PackageMap->GetObjectFromEntityId(EntityId));
+	if (EntityActor != nullptr)
```

Comment:
What's the reason for moving this out of the if statement?

Reply:
Put back.